packages/overlay/src/integrations/sentry/index.ts

```diff
@@ -43,13 +44,25 @@ export default function sentryIntegration(options?: SentryIntegrationOptions) {
     processEvent: (event: RawEventContext) => processEnvelope(event),
 
     tabs: () => {
-      const errorsCount = sentryDataCache
+      const errors = sentryDataCache
         .getEvents()
         .filter(
           e =>
             e.type != 'transaction' &&
             (e.contexts?.trace?.trace_id ? sentryDataCache.isTraceLocal(e.contexts?.trace?.trace_id) : null) !== false,
-        ).length;
+        );
+
+      const errorsCount = errors.length;
+      if (options?.showRecentError && errorsCount) {
```

Comment:
```suggestion
      if (errorsCount && options?.showRecentError) {
```

Cheaper (and easier on the eyes & mind) to check the count first.